### ISSUE TITLE
README enhancement for repository type git, URL

### ISSUE THREAD

**Issue by TonyGravagno** [zokugun/vscode-sync-settings#57]
The README provides a guide for the Git URL:

> url: git@github.com:username/settings.git

For GitHub repos, I propose adding a suggestion to use the generated repo HTTPS URL:

> url: https://github.com/username/reponame.git

If a Personal Access Token is required, a prompt dialog should open.

**Comment by daiyam** [comment]
> No authentifications are stored.
But the git command on your system will need to be able to read/write on the remote repository.

https://github.com/zokugun/vscode-sync-settings?tab=readme-ov-file#remote-git

**Comment by TonyGravagno** [comment]
Yes, but if the "git@" syntax fails it would be helpful to simply suggest that the HTTPS URL will work. A tiny tip like this easily leads to success and costs nothing to add.
